src/main/resources/config.yml

```diff
@@ -1,2 +1,4 @@
 options:
-  auto-update: true
\ No newline at end of file
+  auto-update: true
+  need-wrench-to-break-machines: true
```

Comment:
Rename this

Reply:
To?

Reply:
Something better then this no clue

Reply:
How long can I go? `machine-block-dropped-when-wrench-not-used-to-break-machine` is kinda long

Reply:
Go with something shorter but maybe give info about it

Reply:
Im open to suggestions :c i cant think of anything better than what I have now

Reply:
Lets keep it like this then